isb_web/main.py

```diff
@@ -269,6 +270,25 @@ async def get_solr_select(request: fastapi.Request):
     return isb_solr_query.solr_query(params)
 
 
+@app.post(f"/{THING_URL_PATH}/reliquery", response_model=ReliqueryResponse)
+def get_reliquery(request: fastapi.Request, params: ReliqueryParams) -> ReliqueryResponse:
+    solr_response_dict = isb_solr_query.reliquery_solr_query(params.query)
+    json_response = solr_response_dict.get("response")
+    docs = json_response.get("docs")
+    identifiers = []
+    for dict in docs:
+        identifiers.append(dict["id"])
+    reliquery_response = ReliqueryResponse(
+        query=params.query,
+        identifiers=identifiers,
+        timestamp=datetime.datetime.now().strftime(SOLR_TIME_FORMAT),
+        count=json_response.get("numFound"),
+        return_count=len(identifiers),
+        url=str(request.url),
+        description=params.description)
+    return reliquery_response
```

Comment:
I tried to make it so you could just include the old response in the payload for this method, but that seemed to break if the old response wasn't present.  I'll need to mess around with it a bit more tomorrow.